py/src/call.py

```diff
@@ -163,7 +172,12 @@ def report_failed(logger, command, e):
     logger.trace(f"{command}")
     # we cannot log errors at higher than info, which is the default level, to
     # allow opting in to these and not forcing them on everyone
-    logger.debug(str(e))
+    if DEV_MODE:
+        import traceback
+
+        logger.debug(traceback.format_exc())
```

Comment:
Seems to work for me with an exception object:

```python
In [4]: try:
   ...:     raise ValueError("blah")
   ...: except ValueError as e:
   ...:     import traceback
   ...:     print(traceback.format_exception(e))
   ...: 
['Traceback (most recent call last):\n', '  File "<ipython-input-4-e9dcce3a521d>", line 2, in <module>\n    raise ValueError("blah")\n', 'ValueError: blah\n']
```

Reply:
Ah, my bad: that's probably only because I was testing with Python 3.10: https://docs.python.org/3/library/traceback.html?highlight=format_exception#traceback.format_exception

Reply:
This is the correct invocation for Python 3.9:

```python
In [5]: try:
   ...:     raise ValueError("blah")
   ...: except ValueError as e:
   ...:     import traceback
   ...:     print(traceback.format_exception(type(e), e, e.__traceback__))
   ...: 
['Traceback (most recent call last):\n', '  File "<ipython-input-5-f98dfd010677>", line 2, in <module>\n    raise ValueError("blah")\n', 'ValueError: blah\n']
```

Reply:
Aand fixed.